src/media/js/buttons.js

```diff
@@ -296,11 +296,16 @@ define('buttons',
     }
 
     z.page.on('click', '.mkt-app-button.launch', launchHandler)
-    .on('click', '.mkt-app-button:not(.launch):not(.incompatible)',
+    .on('click', '.mkt-app-button--install:not(.launch):not(.incompatible)',
         _handler(install))
-    .on('click', '.mkt-app-button[disabled]', function(e) {
+    .on('click', '.mkt-app-button--install[disabled]', function(e) {
         e.preventDefault();
         e.stopPropagation();
+    })
+    .on('click', '.mkt-app-button[href]', function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        window.open(this.getAttribute('href'), '_blank');
```

Comment:
We can have a `<button>` but not an `<a>`? [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) says `<a>` can have "flow content" of which `<a>` is a member (as is `<button>`).